CHANGELOG.txt

```diff
@@ -1,3 +1,8 @@
+v5.0.1 (2018-??-??)
```

Comment:
As we are in alpha all we need to do is bump the alpha version, not the patch version, so this should stay under 5.0.0.

Bumping the tedious version in a patch release would be a big nono

Reply:
These all make sense to me. I'll bump it back to v5.0.0